### PR TITLE
GrantsDAO Subgraph

### PR DESCRIPTION
**Pull request by sistemico**
This pull request is to merge the subgraph made for the new GrantsDAO DApp

### CODE REVIEW

**Comment by dvd-schwrtz** [comment]
This looks really good! Did you deploy this to your own personal subgraph first to make sure it is working properly? @justinjmoses will want to look at this but he is off the next couple of days. I suspect it will be merged early next week. If you need to access it in the meantime you can deploy this on your own personal subgraph and temporarily point any dapps to that subgraph link until it is merged. Nice work. Very well done

**Comment by cmalfesi** [comment]
Hi @dvd-schwrtz. Thanks for your feedback! This PR is part of the work that we have done in the [grant](https://github.com/Synthetixio/snx-grants-dao/blob/master/proposals/new-grants-dao-frontend.md). I'm talking with Justin about this. The plan is to merge this and the grantsDAO interface next Monday/Tuesday.

File: subgraphs/synthetix-grantsdao.yaml [hunk 1]
```diff
@@ -0,0 +1,45 @@
...
+      address: '0x86626e1bbbd0ce95ed52e0c5e19f371a6640b591'
+      abi: GrantsDAO
+      startBlock: 9925334
+    mapping:
+      kind: ethereum/events
+      apiVersion: 0.0.4
+      language: wasm/assemblyscript
+      file: ../src/grantsdao-mapping.ts
+      entities:
+        - Test
```
Comment (dvd-schwrtz) [inline subgraphs/synthetix-grantsdao.yaml:20]: Everything looks good. One minor comment is I thought you had to list all the entities here to make them available but if it works without that then I guess it doesn't matter

Comment (justinjmoses) [reply subgraphs/synthetix-grantsdao.yaml:20]: Yeah it can work without them, though probably better to have them

Comment (sistemico) [reply subgraphs/synthetix-grantsdao.yaml:20]: Sure, better to have them in place. Gonna do the change ASAP.

Comment (sistemico) [reply subgraphs/synthetix-grantsdao.yaml]: Done!

**Review by justinjmoses — APPROVED** [review]

**Pull request merged**